ludwig/schema/combiners/common_transformer_options.py

```diff
@@ -6,7 +6,7 @@
 from ludwig.schema.metadata.combiner_metadata import COMBINER_METADATA
 
 
-@dataclass
+@dataclass(repr=False)
```

Comment:
So dataclasses automatically add a `__repr__` that won't be overwritten unless you pass in this `repr=False` parameter. Since the config object is to be exposed in the SDK, we need these custom `__repr__` functions so that when the user runs a cell with the certain section of the config, they can see what's in it.